user_guide_src/source/concepts/autoloader.rst

```diff
@@ -47,25 +47,26 @@ Namespaces
 
 The recommended method for organizing your classes is to create one or more namespaces for your
 application's files. This is most important for any business-logic related classes, entity classes,
-etc. The ``psr4`` array in the configuration file allows you to map the namespace to the directory
+etc. The ``$psr4`` array in the configuration file allows you to map the namespace to the directory
 those classes can be found in:
 
 .. literalinclude:: autoloader/001.php
 
 The key of each row is the namespace itself. This does not need a trailing back slash.
-The value is the location to the directory the classes can be found in. They should
-have a trailing slash.
+The value is the location to the directory the classes can be found in.
 
 .. note:: You can check the namespace configuration by ``spark namespaces`` command::
 
     > php spark namespaces
 
-By default, the application folder is namespace to the ``App`` namespace. While you are not forced to namespace the controllers,
-libraries, or models in the application directory, if you do, they will be found under the ``App`` namespace.
+By default, the application folder is namespace to the ``App`` namespace. You must namespace the controllers,
+libraries, or models in the application directory, and they will be found under the ``App`` namespace.
```

Comment:
usage of `folder` or `directory` may need to be consistent, eg: `application folder` used and next used `application directory`

Reply:
Done.